packages/js/components/src/date-time-picker-control/date-time-picker-control.tsx

```diff
@@ -75,20 +77,36 @@ export const DateTimePickerControl: React.FC< DateTimePickerControlProps > = ( {
 		null
 	);
 
+	const displayFormat = useMemo( () => {
```

Comment:
I think the use of `useMemo` here is a bit overkill as we are not doing any expensive parsing, and we are returning a string (so we don't have to worry about new object references).
It might be worth just removing the use of it.

Reply:
Good point, I'll remove that when I rebase and put it up for re-review!